### PR TITLE
Start vfs

### PR DESCRIPTION
**Pull request by jonathandturner**
This is the first few commands for the vfs support: `ls` and `open`. I'm currently hardwiring how we populate the files, but this should give us a start to build towards a nice vfs experience.

I also went ahead and added a simple `sys` command that we can grow later.

### CODE REVIEW

File: www/index.js [hunk 1]
```diff
@@ -110,6 +110,9 @@ examplesContainer.addEventListener("click", (event) => {
 nuinput.addEventListener("keydown", (event) => {
   if (event.key == "Enter") {
     if (event.metaKey || event.ctrlKey) {
+      event.preventDefault();
+      runCommand();
```
Comment (jzaefferer) [inline www/index.js:114]: Why did you change this? I reverted the behaviour on purpose, as the help text ("Press Ctrl+Enter (or Command+Enter on OSX) to add a new line (add linebreak)") indicates.

File: www/index.html [hunk 2]
```diff
@@ -1,56 +1,85 @@
...
-      .output-error {
-        font-family: monospace;
-      }
-      .output-error--inline {
-        color: red;
-      }
-      td {
-        min-width: 6rem;
+
+<head>
```
Comment (jzaefferer) [inline www/index.html:4]: What code formatter are you using? I'm using prettier, which has different defaults than this.

File: www/index.html [hunk 3]
```diff
@@ -1,56 +1,85 @@
...
+
+  <script type="text/javascript" src="browserfs.min.js"></script>
+  <script type="text/javascript">
+    // Installs globals onto window:
+    // * Buffer
+    // * require (monkey-patches if already defined)
+    // * process
+    // You can pass in an arbitrary object if you do not wish to pollute
+    // the global namespace.
+    BrowserFS.install(window);
```
Comment (jzaefferer) [inline www/index.html:46]: Is there a reason for this being inline in the html file? If not, can we move it to bootstrap.js? In there we could then move the `import()` call into the `BrowserFS.configue` callback.

File: www/module.js [hunk 4]
```diff
@@ -5,3 +5,48 @@ function getRandomInt(max) {
...
+
+    let output = JSON.stringify(contents);
+    return output;
+}
+
+export function readdir(path) {
+    let results = [];
+
+    var fs = BrowserFS.BFSRequire('fs');
+    fs.readdir(path, function (e, contents) {
```
Comment (jzaefferer) [inline www/module.js:28]: Is this really sync? Or is there a readdirSync variant, like `readFileSync`?

Comment (jzaefferer) [reply www/module.js:28]: => https://github.com/nushell/wasm-nu/pull/23

File: www/index.js (hunk 1 above)
Comment (jzaefferer) [reply www/index.js:114]: I just got used to being able to simply hit enter to run a command 🙈 No I have to unlearn that again

File: www/index.html (hunk 3 above)
Comment (jzaefferer) [reply www/index.html:46]: => https://github.com/nushell/wasm-nu/pull/24

File: www/index.js (hunk 1 above)
Comment (jonathandturner) [reply www/index.js:114]: Oooooh. I misread the logic. So you wanted ctrl+enter to add a newline not to run the command? Sorry about that. I had thought you wanted ctrl+enter to run the command so I switched it. We can switch it back.

File: www/index.html (hunk 2 above)
Comment (jonathandturner) [reply www/index.html:4]: I use vscode, so I think it's whatever formatter it uses by default. Should I change it?

Comment (jzaefferer) [reply www/index.html:4]: I suggest we use https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode to get consistent output. We could add prettier as a CI tool for linting later.
